examples/pytorch/text-classification/run_glue.py

```diff
@@ -536,7 +536,9 @@ def train_func(model):
         if quantizer.approach == LpotQuantizationMode.DYNAMIC.value:
             q_model = quantizer.fit_dynamic()
         elif quantizer.approach == LpotQuantizationMode.STATIC.value:
-            quantizer.calib_dataloader = trainer.get_eval_dataloader()
+            from optimum.intel.lpot.utils import LpotDataloader
+            eval_dataloader = trainer.get_eval_dataloader()
+            quantizer.calib_dataloader = LpotDataloader(eval_dataloader)
             q_model = quantizer.fit_static()
```

Comment:
Maybe we can wrap the dataloader in LpotQuantizer.
@calib_dataloader.setter
 def calib_dataloader(self, dataloader: DataLoader):
    from optimum.intel.lpot.utils import LpotDataloader
    self._calib_dataloader = LpotDataloader(dataloader)

Reply:
Good point, thanks for the suggestion !